src/App.tsx

```diff
@@ -1,30 +1,13 @@
 import React, { useState, useEffect } from 'react';
 import { Route, BrowserRouter, Routes, useParams } from 'react-router-dom';
-import Game from './interfaces/Game'
 import LibrariesPage from './pages/LibrariesPage';
 import LibraryPage from './pages/LibraryPage';
 import GamePage from './pages/GamePage';
 import GameInstancePage from "./pages/GameInstancePage";
 import UserProfilePage from './pages/UserProfilePage';
+import RegistrationPage from './pages/RegistrationPage';
 
 const App = () => {
-  let emptyGame: Game = { id: "", name: "", platforms: [], genres: [], franchises: [], companies: [], releaseDate: [], summary: "", images: []}
```

Comment:
This keeps causing the JSON parsing issue - we're not using it anyway, so removing